hathor/transaction/resources/block_at_height.py

```diff
@@ -75,8 +75,29 @@ def render_GET(self, request: 'Request') -> bytes:
             })
 
         block = self.manager.tx_storage.get_transaction(block_hash)
-
         data = {'success': True, 'block': block.to_json_extended()}
+
+        # include_transactions parameter is optional and must be an integer
+        try:
+            include_transactions = parse_int(args.get('include_transactions', 0))
```

Comment:
I think we should add a query with this parameter to the OpenAPI definition.

Reply:
Fixed. Thanks!